inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/range_ref.cl

```diff
@@ -0,0 +1,11 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+KERNEL (range_ref)(const __global INPUT0_TYPE *startP, const __global INPUT1_TYPE *stopP,
```

Comment:
Just for better understanding: is it correct that you have stopP argument but don't use it inside your kernel code?

Reply:
stop is needed to calculate size of output vector, kernel itself has no use for it. i didn't check what it would take to stop upper layers from passing it.

Reply:
got rid of `stopP`